### PR TITLE
fix(pointers): [iOS] Add support of proper bubbling of handled events

### PR DESCRIPTION
**Pull request by dr1rrb**
fixes https://github.com/unoplatform/nventive-private/issues/220

## Bugfix / Feature
Add support of proper bublling of handled events

## What is the current behavior?
If a control handles the pressed event, parents won't receive any subsequent event

## What is the new behavior?
We no longer prevent bubbling of native pressed event, we instead filter them out on parents

## PR Checklist
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/SamplesApp/UITests.Shared/Windows_UI_Input/PointersTests/NestedHandling.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Uno.UI.Samples.Controls;
+
+namespace UITests.Windows_UI_Input.PointersTests
+{
+	[Sample(
+		"Pointers",
+		Description = "Press the pink square, move a bit and release while remaining on the pink square.")]
```
Comment (davidjohnoliver) [inline src/SamplesApp/UITests.Shared/Windows_UI_Input/PointersTests/NestedHandling.xaml.cs:21]: Is it possible to do this in an automated test?

**Review by davidjohnoliver — APPROVED** [review]

File: src/SamplesApp/UITests.Shared/Windows_UI_Input/PointersTests/NestedHandling.xaml.cs (hunk 1 above)
Comment (dr1rrb) [reply src/SamplesApp/UITests.Shared/Windows_UI_Input/PointersTests/NestedHandling.xaml.cs]: Yep it was already in the pipeline ;)

**Review by ajpinedam — APPROVED** [review]

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/NestedHandling_Tests.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	[TestFixture]
+	public partial class NestedHandling_Tests : SampleControlUITestBase
+	{
+		private const string _sample = "UITests.Windows_UI_Input.PointersTests.NestedHandling";
+
+		[Test]
+		[AutoRetry]
+		public void When_NestedHandlesPressed_Then_ContainerStillGetsSubsequentEvents()
+		{
+			Run(_sample);
```
Comment (jeromelaban) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/NestedHandling_Tests.cs:21]: @dr1rrb the test is not running under iOS :/

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/3.7

**Pull request merged**